metsrw/utils.py

```diff
@@ -20,6 +35,13 @@ def lxmlns(arg):
 
 
 # CONSTANTS
-
 FILE_ID_PREFIX = 'file-'
 GROUP_ID_PREFIX = 'Group-'
+
+
+# URL helpers
+def urlencode(url):
+    parsed = urlparse.urlparse(url)
+    for attr in ('path', 'params', 'query', 'fragment'):
+        parsed = parsed._replace(**{attr: quote_plus(getattr(parsed, attr), safe='/')})
```

Comment:
It seems like you've been doing an effort in this PR to keep columns down to 79; FYI this one has 87.